server.go

```diff
@@ -99,12 +100,22 @@ func rpcHandler(w http.ResponseWriter, r *http.Request) {
 func mainHandler(w http.ResponseWriter, r *http.Request) {
 	path := r.URL.Path[1:]
 	log.Printf("Serving file: %s", path)
-	if !isLoggedIn(r) {
+	userIsLoggedIn, userID := isLoggedIn(r)
+	if !userIsLoggedIn {
 		appPath := fmt.Sprintf("login/%s", path)
 		http.ServeFile(w, r, appPath)
 	} else {
-		appPath := fmt.Sprintf("app/%s", path)
-		http.ServeFile(w, r, appPath)
+		var err error
+		var isRoot, _ = getIsRoot(userID)
+		log.Printf("isRoot: %t", isRoot)
+		checkErr(err)
+		if  isRoot {
```

Comment:
This is incorrect. The handler is in admin, not here. the main handler is only for normal users